package/doc/sphinx/source/documentation_pages/units.rst

```diff
@@ -1,2 +1,27 @@
 .. automodule:: MDAnalysis.units
 
+
+
+.
+
+
+Changes made in this Pull Request:
+
+- Fixed some typos : 
+
+
+> - units and conversions [page](https://docs.mdanalysis.org/2.1.0-dev0/documentation_pages/units.html?highlight=unit%20conversion#conversions), `MDAnalysis.units.timeUnit_factor`: fix scientific notation in **time unit** = 4.888821`E-14` sec to 4.888821`e-14` sec
+
+
+
+
+> - units and conversions [page](https://docs.mdanalysis.org/2.1.0-dev0/documentation_pages/units.html?highlight=unit%20conversion#conversions), `MDAnalysis.units.densityUnit_factor`: fix scientific notation in **Example for how to calculate the conversion factor** where `Å−3 `  to  ` A**-3 `
+
+
+
+
+
+.
+
+
+
```

Comment:
This addition to the documentation looks like this: https://mdanalysis--3639.org.readthedocs.build/en/3639/documentation_pages/units.html#references-and-footnotes

We do not usually want the issue comments in the documentation. The problem you are fixing should only be described in the PR, i.e. at this URL/HTML page, instead of being added to the code or documentation itself. Please remove this.